language-support/ts/daml-ledger/index.test.ts

```diff
@@ -72,6 +72,7 @@ jest.mock('isomorphic-ws', () => class {
 
 
 const Foo: Template<Foo, string, "foo-id"> = {
+  sdkVersion: '0.0.0-SDKVERSION',
```

Comment:
This is going to break on release builds where the version is not `0.0.0`. Try setting `DAML_SDK_RELEASE_VERSION` to test this.

Reply:
Looks like this works fine. Tested with different `DAML_SDK_RELEASE_VERSION` and `bazel run //language-support/ts/daml-ledger:test`